src/main/kotlin/com/example/mykku/admin/service/AdminBoardService.kt

```
@@ -0,0 +1,39 @@
+) {
+
+    @Transactional(propagation = Propagation.NOT_SUPPORTED)
+    fun create(request: BoardCreateRequest): BoardResult {
+        val logoUrl = imageUploadService.uploadImage(request.logo).url
+        val command = CreateBoardCommand(
+            title = request.title,
+            logo = logoUrl
+        )
+        return try {
+            createBoardUseCase.create(command)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)